package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cycle-drag-and-drop",
+  "version": "1.0.0",
+  "description": "Drag and drop in Cycle",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "budo index.js:bundle.js -- -t babelify | garnish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/raquelxmoss/cycle-drag-and-drop.git"
+  },
+  "author": "Raquel Moss",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/raquelxmoss/cycle-drag-and-drop/issues"
+  },
+  "homepage": "https://github.com/raquelxmoss/cycle-drag-and-drop",
+  "dependencies": {
+    "@cycle/core": "^7.0.0-rc8",
+    "@cycle/dom": "^10.0.0-rc34",
+    "@cycle/xstream-run": "^3.0.2",
+    "xstream": "^5.0.6"
```

Comment:
I pulled down the app, ran `npm install` and tried to start it but `lodash` is missing. It's good practice to always `--save` or `--save-dev` your packages so that your app is always installable.